blueprint-reference: update file customization paths

lzap opened the pull request:
A docs update for:

https://github.com/osbuild/images/pull/528

Note in the PR I also propose `/var/tmp` but despite this gets through or not, I would not recommend documenting that. This would be a "workaround" path.

ochosi reviewed (approved):
Agreed. In its current form, the news link makes no sense.

We could generate the news page with each GitHub release creating an entry. Feels like duplication though.

ochosi commented:
@supakeen Could you get CI to pass on this PR and on #58 ?

Would be great to get both merged, but it seems there are dangling references.

lzap committed:
blueprint-reference: update file customization paths

A docs update for:

https://github.com/osbuild/images/pull/528

Note in the PR I also propose `/var/tmp` but despite this gets through or not, I would not recommend documenting that. This would be a "workaround" path.

ondrejbudai commented on docs/user-guide/01-blueprint-reference.md:
```diff
@@ -629,11 +629,9 @@ The service names are systemd service units. You may specify any systemd unit fi
-When using the custom files and directories customization, the following rules apply:
-
-- The path must be an absolute path and must be under `/etc` or `/root`.
+- The path must be an absolute path and must be under `/etc`, `/root` or `/usr/local`.
```
Note that this is wrong, the current policy ONLY custom files (no dirs) and ONLY under `/usr/local/bin` and `/usr/local/sbin`. @lzap This might be worth fixing.

Note that https://github.com/osbuild/images/pull/540 is currently open which might change the policies once again.

lzap replied:
Sure: https://github.com/osbuild/osbuild.github.io/pull/63
